package/gluon-web-admin/luasrc/lib/gluon/web/model/admin/remote.lua

```diff
@@ -86,6 +94,11 @@ local function set_password(password)
 end
 
 function f_password:write()
+	if pw1.data:len() < min_password_length then
```

Comment:
Using the datatype is much better than checking manually, as the fields will give direct feedback (background color) then. Nicer textual error messages are a TODO for future improvements in gluon-web.

We're only using the ugly errmessage because we can't express the "same value" constraint using datatypes, ideally, we'd change that to use a similar validation in the future, too.